### PR TITLE
refactor(p-tooltip): change from `stories.ts` to `stories.mdx`

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skipping review is not a problem (```style```, ```chore``` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [ ] Feature improvement
  - [ ] These changes require a usage change
- [x] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] Updated Storybook documents
- [ ] Tested with console

### Description
SSIA
스크린샷 2022-10-26 오후 3 53 16

### CODE REVIEW

File: src/data-display/tooltips/story-helpers.ts [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        },
+        control: {
+            type: 'select',
+            options: [...Object.values(POSITIONS)],
+        },
+    },
+    options: {
+        name: 'options',
+        type: { name: 'object' },
+        description: 'Position of tooltip.',
```
Comment (song2kim) [inline src/data-display/tooltips/story-helpers.ts]: Please check if the description is correct (description 이 맞는지 확인부탁드려요)

**Review by song2kim — APPROVED** [review]
LGTM I left a comment

File: src/data-display/tooltips/story-helpers.ts (hunk 1 above)
Comment (yuda110) [reply src/data-display/tooltips/story-helpers.ts]: thanks!

**Pull request merged**